app/views/hq/sponsors/_sponsorships_active.html.haml

```diff
@@ -0,0 +1,47 @@
+- if sponsorships.empty?
+  .panel.panel-default
+    .panel-body
+      No currently sponsored orphans
+- else
+  .table-responsive
+    %table.table
+      .table_title
+        = "#{sponsorships.count} currently sponsored #{'orphan'.pluralize(sponsorships.count)}"
+      %thead
+        %tr
+          %th
+            Orphan name
+          %th
+            Orphan date of birth
+          %th
+            Orphan gender
+          %th
+            Sponsorship Began
+          %th
+
+          %th
+
+      %tbody
+        - sponsorships.each do |sponsorship|
+          %tr
+            %td
+              = link_to sponsorship.orphan.name, hq_orphan_path(sponsorship.orphan)
+            %td
+              = sponsorship.orphan.date_of_birth
+            %td
+              = sponsorship.orphan.gender
+            %td
+              = sponsorship.start_date
+            %td
+              =form_for(sponsorship,
+                  :url => {controller: :sponsorships, action: :inactivate, sponsor_id: sponsorship.sponsor_id, id: sponsorship.id},
+                  method: :put) do |f|
+                =f.submit "End Sponsorship", class: "btn btn-default"
+                =f.label :end_date, "on"
+                =f.text_field :end_date, value: Date.current
+            %td
+              =link_to "Delete",
```

Comment:
I originally had it as 'X' because of a). space; and b). sponsorship deletion is something that should not be a part of the standard workflow. The client requested it as an option for when they mess up sponsorship creation. So, I wanted deletion to be obscure, dangerous-looking and maybe convoluted somehow, to really send the message _"Are you sure?!!!"_.

If we decide to go the 'X' route, we can use Bootstrap's 'glyphicon-remove' coloured blood-red, poisonous-orange or something. @christophermlne, does that make you cringe? :)